### ISSUE TITLE
Color contrast between "Search" icon and its background is less than 3:1: A11y_VisualStudioCodeServices_Uninstall Visual Studio Code_Search icon_ColorContrast.

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-docs#7417]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

## GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #ChromiumEdge; #BM_VisualStudioCodeServices_Web_July2024; #Visual Studio Code Services; #Benchmark; #Win11; #FTP; #Desktopweb; #WCAG1.4.3; #ContrastMinimum; #Closed; #Regressed:06-28-24

## Environment Details: 
Application Name:  Visual Studio Code Services
URL: https://code.visualstudio.com/docs/setup/uninstall?dark-plus-v2#_windows
Microsoft Edge version 126.0.2592.68 (Official build) (64-bit)

## Repro steps:
1. Launch the application URL: https://code.visualstudio.com/docs/setup/uninstall?dark-plus-v2#_windows and press ENTER.
2. "Uninstall Visual Studio Code" screen gets opened.
3. TAB to "Search field" in the header.
4. Turn on accessibility insights for desktop and check color contrast between "Search icon" and background.

## Actual:
Color contrast between "Search" icon and its background is 2.532:1 which is less than the required color contrast ratio 3:1.

Note: Issue is repro with below URLs as well
https://code.visualstudio.com/docs/setup/uninstall?dark-plus-v2
https://code.visualstudio.com/docs/supporting/faq?dark-plus-v2
https://code.visualstudio.com/docs?dark-plus-v2
https://code.visualstudio.com/download?dark-plus-v2
https://code.visualstudio.com/insiders/?dark-plus-v2
https://code.visualstudio.com/?dark-plus-v2
https://code.visualstudio.com/docs/python/editing?dark-plus-v2
https://code.visualstudio.com/docs/python/jupyter-support-py?dark-plus-v2
https://code.visualstudio.com/docs/python/linting?dark-plus-v2
https://code.visualstudio.com/docs/remote/remote-overview?dark-plus-v2
https://code.visualstudio.com/docs/remote/troubleshooting?dark-plus-v2
https://code.visualstudio.com/docs/remote/wsl?dark-plus-v2
https://code.visualstudio.com/docs/?dv=win64user&dark-plus-v2
https://code.visualstudio.com/docs/editor/command-line?dark-plus-v2
https://code.visualstudio.com/docs/getstarted/introvideos?dark-plus-v2
https://code.visualstudio.com/docs/getstarted/keybindings?dark-plus-v2
https://code.visualstudio.com/docs/getstarted/locales?dark-plus-v2
https://code.visualstudio.com/docs/java/java-debugging?dark-plus-v2
https://code.visualstudio.com/docs/languages/csharp?dark-plus-v2
https://code.visualstudio.com/docs/python/debugging?dark-plus-v2

## Expected
Color contrast between "Search" icon and its background should be greater than or equal to 3:1

## User Impact
Low vision users will not be able to access search icon button if Color contrast between "Search" icon and its background is less than expected.

## Attachment
![aaaaaaaaaaaa](https://github.com/microsoft/vscode-docs/assets/98821792/237fe983-ba51-4087-aadb-41023239d217)

![1 note cc](https://github.com/microsoft/vscode-docs/assets/98821792/b01312a6-ee88-4406-a96b-3407e7073d2f)

![Screenshot 2024-06-26 184242](https://github.com/microsoft/vscode-docs/assets/98821792/4edf6af3-74b6-4ec7-a1ba-925ff1b5b450)

**Comment by daviddossett** [comment]
Fixed via https://github.com/microsoft/vscode-website/pull/1801

![CleanShot 2024-06-26 at 14 23 19@2x](https://github.com/microsoft/vscode-docs/assets/25163139/0b4c255c-d8cd-4900-9884-7a566a770190)

**Comment by 12-shweta** [comment]
#rev:shwarh;

**Comment by 12-shweta** [comment]
#A11yPreview;

**Comment by MounicaDantuluri0809** [comment]
@daviddossett 

Issue not fixed. Search Button is now not even visible. Please find below for reference of current page UI in "https://code.visualstudio.com/?dark-plus-v2" environment

@prabhudevu FYI.

![Issue7417_Not Fixed](https://github.com/microsoft/vscode-docs/assets/136050453/75c934d0-8b78-40a8-9dbb-b90f098b2df7)

**Comment by daviddossett** [comment]
Hm, seems like it's not rendering at all. I'll get a fix going

**Comment by daviddossett** [comment]
Will be fixed via https://github.com/microsoft/vscode-website/pull/1821

![Image](https://github.com/user-attachments/assets/99d829e1-2ea9-4f23-a7e6-0605ec0f83e3)

![Image](https://github.com/user-attachments/assets/0bca513d-a751-4aa6-bc1f-65b89c92313b)

**Comment by prabhudevu** [comment]
I have verified the issue in the ENV: https://code.visualstudio.com/?dark-plus-v2

Issue is fixed. closing the bug
